### ISSUE TITLE
Zellij crashed

### ISSUE THREAD

**Issue by tafia** [zellij-org/zellij#2834]
**Basic information**

`zellij --version`: zellij 0.38.1
`stty size`: 122 307
`uname -av` or `ver`(Windows): Linux 5.15.90.1-microsoft-standard-WSL2 #1 SMP Fri Jan 27 02:56:13 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

List of programs you interact with as:
`hx --version`: helix 23.05 (16e0340b)
`docker --version`: Docker version 24.0.5, build ced0996
`alacritty --version`: alacritty 0.12.0 (5a728195) - alacritty installed on windows to launch wsl2

**Further information**

Nothing out of the ordinary. I had 4 tabs opened ~2-3 panes in each.
Was running 2 docker commands at once (build and pull).

**Logs**

```sh
$ bat -p /tmp/zellij-1000/zellij-log/zellij.log
INFO   |zellij_client            | 2023-10-03 09:35:30.676 [main      ] [zellij-client/src/lib.rs:152]: Starting Zellij client!
INFO   |zellij_server            | 2023-10-03 09:35:30.685 [main      ] [zellij-server/src/lib.rs:245]: Starting Zellij server!
WARN   |zellij_server::route     | 2023-10-03 09:35:30.747 [server_router] [zellij-server/src/route.rs:837]: Server not ready, trying to place instruction in retry queue...
WARN   |zellij_server::route     | 2023-10-03 09:35:30.747 [server_router] [zellij-server/src/route.rs:907]: Server ready, retrying sending instruction.
WARN   |zellij_server::route     | 2023-10-03 09:35:30.747 [server_router] [zellij-server/src/route.rs:837]: Server not ready, trying to place instruction in retry queue...
WARN   |zellij_server::route     | 2023-10-03 09:35:30.747 [server_router] [zellij-server/src/route.rs:837]: Server not ready, trying to place instruction in retry queue...
WARN   |zellij_server::route     | 2023-10-03 09:35:30.747 [server_router] [zellij-server/src/route.rs:907]: Server ready, retrying sending instruction.
WARN   |zellij_server::route     | 2023-10-03 09:35:30.747 [server_router] [zellij-server/src/route.rs:837]: Server not ready, trying to place instruction in retry queue...
WARN   |zellij_server::route     | 2023-10-03 09:35:30.747 [server_router] [zellij-server/src/route.rs:837]: Server not ready, trying to place instruction in retry queue...
WARN   |zellij_server::route     | 2023-10-03 09:35:30.747 [server_router] [zellij-server/src/route.rs:907]: Server ready, retrying sending instruction.
WARN   |zellij_server::route     | 2023-10-03 09:35:30.747 [server_router] [zellij-server/src/route.rs:837]: Server not ready, trying to place instruction in retry queue...
WARN   |zellij_server::route     | 2023-10-03 09:35:30.747 [server_router] [zellij-server/src/route.rs:837]: Server not ready, trying to place instruction in retry queue...
INFO   |zellij_server            | 2023-10-03 09:35:30.748 [main      ] [zellij-server/src/lib.rs:913]: Compiling plugins using Cranelift
INFO   |zellij_server::plugins   | 2023-10-03 09:35:30.749 [wasm      ] [zellij-server/src/plugins/mod.rs:137]: Wasm main thread starts
INFO   |zellij_server::plugins::p| 2023-10-03 09:35:30.790 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:487]: Loaded plugin 'tab-bar' from cache folder at '/home/jtuffe/.cache/zellij' in 17.670098ms
INFO   |zellij_server::plugins::p| 2023-10-03 09:35:30.792 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:487]: Loaded plugin 'status-bar' from cache folder at '/home/jtuffe/.cache/zellij' in 18.630298ms
WARN   |zellij_server::route     | 2023-10-03 09:35:35.378 [server_router] [zellij-server/src/route.rs:907]: Server ready, retrying sending instruction.
INFO   |zellij_server::plugins::w| 2023-10-03 09:41:03.026 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 2
INFO   |zellij_server::plugins::w| 2023-10-03 09:41:03.027 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 3
INFO   |zellij_server::plugins::w| 2023-10-03 14:18:22.598 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 8
INFO   |zellij_server::plugins::w| 2023-10-03 14:18:22.599 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 9
INFO   |zellij_server::plugins::w| 2023-10-04 13:26:46.293 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 4
INFO   |zellij_server::plugins::w| 2023-10-04 13:26:46.295 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 5
INFO   |zellij_server::plugins::w| 2023-10-04 13:37:14.646 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 16
INFO   |zellij_server::plugins::w| 2023-10-04 13:37:14.646 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 17
ERROR  |zellij_utils::errors::not| 2023-10-04 13:53:09.899 [screen    ] [zellij-utils/src/errors.rs:614]: Panic occured:
             thread: screen
             location: At zellij-server/src/panes/grid.rs:1390:46
             message: called `Option::unwrap()` on a `None` value
INFO   |zellij_server::plugins::w| 2023-10-04 13:53:09.903 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 0
INFO   |zellij_server::plugins::w| 2023-10-04 13:53:09.907 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 15
INFO   |zellij_server::plugins::w| 2023-10-04 13:53:09.909 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 13
INFO   |zellij_server::plugins::w| 2023-10-04 13:53:09.912 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 12
ERROR  |zellij_server::os_input_o| 2023-10-04 13:53:09.914 [screen    ] [zellij-server/src/os_input_output.rs:867]: Failed to apply cached resizes: failed to send message to pty writer
INFO   |zellij_server::plugins::w| 2023-10-04 13:53:09.914 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 10
INFO   |zellij_server::plugins::w| 2023-10-04 13:53:09.916 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 14
INFO   |zellij_server::plugins::w| 2023-10-04 13:53:09.925 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 1
INFO   |zellij_server::plugins::w| 2023-10-04 13:53:09.928 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 11
INFO   |zellij_server::plugins::w| 2023-10-04 13:53:09.929 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 7
INFO   |zellij_server::plugins::w| 2023-10-04 13:53:09.931 [wasm      ] [zellij-server/src/plugins/wasm_bridge.rs:198]: Bye from plugin 6
INFO   |zellij_server::plugins   | 2023-10-04 13:53:09.934 [wasm      ] [zellij-server/src/plugins/mod.rs:339]: wasm main thread exits
```

**Comment by imsnif** [comment]
Hi @tafia - I am very sorry for the experience! I believe this is fixed in 0.38.2.

**Comment by tafia** [comment]
Don't be sorry! Not a big deal at all, and thanks a lot for zellij, it is a REALLY nice to use!
Installed 0.38.2.

Thanks!
